src/app/components/client/toolbar/UserMenu.tsx

```diff
@@ -108,6 +115,24 @@ export const UserMenu = (props: UserMenuProps) => {
           {l10n.getString("user-menu-settings-label")}
         </Link>
       </Item>
+      {hasPremium(props.user) && (
+        <Item
+          key={itemKeys.contact}
+          textValue={l10n.getString("user-menu-contact-label")}
+        >
+          <a
+            className={styles.menuItemCta}
+            href="https://support.mozilla.org/questions/new/monitor/form"
```

Comment:
Don't feel strongly about this, but should this be a const in https://github.com/mozilla/blurts-server/blob/3db4ec0f20ac2c5c332df7ca9c33bb4163d5d90d/src/constants.ts?

PROS:
- consistency with `href={CONST_URL_SUMO_MONITOR_SUPPORT}` below on L142

CONS:
- one more level of indirection

Reply:
Well, you called it :sweat_smile: I was in doubt between avoiding the indirection and being consistent, and went with avoiding indirection, with someone calling it out in reviews being the decider. You did, so I changed it in 7d3787c73.